src/extensions/renderer/canvas/drawing-edges.js

```diff
@@ -208,8 +208,18 @@ CRp.drawEdgePath = function( edge, context, pts, type ){
       case 'straight':
       case 'segments':
       case 'haystack':
-        for( let i = 2; i + 1 < pts.length; i += 2 ){
-          context.lineTo( pts[ i ], pts[ i + 1] );
+        const taxiRadius= edge.pstyle( 'taxi-radius' )
+        const taxiRadiusValue = taxiRadius ? taxiRadius.value : 0
```

Comment:
```suggestion
        const taxiRadiusValue = taxiRadius ? taxiRadius.pfValue : 0
```

Reply:
Preferred value (`pfValue`, i.e. px), not raw value (`value`, e.g. `1em` has `value` of `1`)